src/main/java/com/hms_networks/americas/sc/extensions/historicaldata/HistoricalDataEbdRequest.java

```diff
@@ -0,0 +1,243 @@
+package com.hms_networks.americas.sc.extensions.historicaldata;
+
+import com.hms_networks.americas.sc.extensions.system.time.SCTimeUnit;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * Utility methods for building EBD requests for historical data according to Export Block
+ * Descriptor REFERENCE GUIDE RG-0009-00-EN, Publication date 30/08/2023
+ *
+ * @author HMS Americas FAE team
+ * @version 1.0
+ * @since 1.16.0
+ */
+public class HistoricalDataEbdRequest {
+  /** EBD syntax for group filter */
+  private static final String EBD_FIELD_GROUP_FILTER = "$fl";
+
+  /** EBD syntax for data type */
+  private static final String EBD_FIELD_DATA_TYPE = "$dt";
+
+  /** EBD syntax for data type historical log */
+  private static final String EBD_FIELD_DATA_TYPE_HISTORICAL_LOG = "HL";
+
+  /** EBD syntax for data type historical string */
+  private static final String EBD_FIELD_DATA_TYPE_HISTORICAL_STRING = "HS";
+
+  /** EBD syntax for start time */
+  private static final String EBD_FIELD_START_TIME = "$st";
+
+  /** EBD syntax for start time */
+  private static final String EBD_FIELD_END_TIME = "$et";
+
+  /** EBD syntax for respond format text */
+  private static final String EBD_FIELD_FORMAT_AS_TEXT = "$ftT";
+
+  /** EBD syntax for timestamp type */
+  private static final String EBD_FIELD_TIMESTAMP = "$ts";
+
+  /** EBD syntax for Flexy should keep last point in memory for subsequent historical data reads */
+  private static final String EBD_FIELD_UPDATE_TIME = "$ut";
+
+  /**
+   * Get EBD request string for timestamp type.
+   *
+   * @param exportDataInUtc export data with time string as ISO 8601 UTC format if {@code true}
+   *     (instead of local)
+   * @return formatted time string for EBD calls
+   */
+  private static String getEBDTimestampType(boolean exportDataInUtc) {
```

Comment:
Several methods in this class violate the Google Java Format method naming conventions. `EBD` should be `Ebd`